shared/provision/password/container.tsx

```diff
@@ -15,8 +17,11 @@ const mapStateToProps = state => ({
 })
 
 const mapDispatchToProps = dispatch => ({
+  _onForgotPassword: username =>
+    flags.resetPipeline
+      ? dispatch(RecoverPasswordGen.createStartRecoverPassword({username}))
```

Comment:
Something should cancelling provisioning; either here or in the saga. I completed the flow then got this:
<img width="592" alt="image" src="https://user-images.githubusercontent.com/11968340/65461549-25721a80-de22-11e9-96c7-e1428cc77647.png">